viewer-prototype/src/browser/trace-viewer/components/abstract-output-component.tsx

```diff
@@ -17,11 +17,17 @@ export interface AbstractOutputProps {
     resolution?: number;
     outputDescriptor: OutputDescriptor;
     style: OutputComponentStyle;
+    widthWPBugWorkaround: number; // WidthProvider has a bug. Workaround for it needs width to be explicitly passed
```

Comment:
Could you expand on the bug, link to it, tie it to a specific version of a package, just so we know next year if this can be changed.

Reply:
https://github.com/STRML/react-grid-layout/issues/961. Is it Ok to have hyperlinks in code comments?